components/overloop/actions/mark-deal-won/mark-deal-won.mjs

```diff
@@ -0,0 +1,27 @@
+import overloop from "../../overloop.app.mjs";
+
+export default {
+  key: "overloop-mark-deal-won",
+  name: "Mark Deal as Won",
+  description: "Marks a deal as won. [See the docs](https://apidoc.overloop.com/#mark-as-won)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    overloop,
+    dealId: {
+      propDefinition: [
+        overloop,
+        "dealId",
+      ],
+    },
+  },
+  async run({ $ }) {
+    const { data: response } = await this.overloop.markDealLost(this.dealId, {
```

Comment:
is that correct? this action is `mark-deal-won` but the method called is `markDealLost`